web/src/components/organisms/Project/Content/ContentList/hooks.ts

```diff
@@ -32,6 +32,9 @@ export default () => {
       title: field.title,
       dataIndex: ["fields", field.id],
       key: field.id,
+      width: 128,
+      minWidth: 128,
```

Comment:
Unless there is a weird issue with AntD, `width` should be enough, no?

Reply:
@KaWaite , `minWidth` is important for draggable so user can't drag the column less than the defined `minWidth`, by default and in antD example user can, but Red suggested to have this min width.

Reply:
Ahhh ok!